props/saber_fett263_buttons.h

```diff
@@ -1418,8 +1425,12 @@ SaberFett263Buttons() : PropBase() {}
     itoa(Color16(color_source).g, new_color + strlen(new_color), 10);
     strcat(new_color, ",");
     itoa(Color16(color_source).b, new_color + strlen(new_color), 10);
-    current_preset_.SetStyle(blade,style_parser.SetArgument(current_preset_.GetStyle(blade), effect + 2, new_color));
-    if (color_mode_ != CC_COLOR_LIST) color_mode_ = NONE;
+    if (color_mode_ == CC_COLOR_LIST) {
+      ONCEPERBLADE(NEW_COLOR_ALL_BLADES);
```

Comment:
There is really no need to use ONCEPERBLADE here. Since your macro doesn't use ## to concatenate N with anything you can just use a loop from 1 to NUM_BLADES instead.

Reply:
replaced with while()